addons/stock_dropshipping/models/purchase.py

```diff
@@ -31,6 +31,13 @@ def _prepare_stock_moves(self, picking):
         res = super(PurchaseOrderLine, self)._prepare_stock_moves(picking)
         for re in res:
             re['sale_line_id'] = self.sale_line_id.id
+            if self.order_id.dest_address_id:
+                # In a dropshipping context we do not need the description of the purchase order or it will be displayed
+                # in Delivery slip report and it may be confusing for the customer to see several times the same text (product name + description_picking).
+                product = self.product_id.with_context(lang=self.order_id.dest_address_id.lang or self.env.user.lang)
```

Comment:
Shouldn't be the lang of the Sales order's partner instead ?

Reply:
It is related to https://github.com/odoo/odoo/pull/179104#discussion_r1753173253 IMO use `dest_address_id` is correct in the dropshipping context